reana_job_controller/job_monitor.py

```diff
@@ -312,6 +313,7 @@ def watch_jobs(self, job_db, app=None):
                         slurm_jobs[job_dict['backend_job_id']] = id
                 if not slurm_jobs.keys():
                     logging.error('No slurm jobs')
+                    time.sleep(120)
```

Comment:
Can you make it a configuration variable instead of hard-coding 120?

Reply:
... Hmm. we could address the time.sleep issue later also for other occurrences:

```console
$ git grep time.sleep
reana_job_controller/htcondorcern_job_manager.py:203:            time.sleep(10)
reana_job_controller/htcondorcern_job_manager.py:213:            time.sleep(10)
reana_job_controller/htcondorcern_job_manager.py:229:            time.sleep(10)
reana_job_controller/htcondorcern_job_manager.py:250:            time.sleep(10)
reana_job_controller/job_monitor.py:259:                time.sleep(120)
reana_job_controller/job_monitor.py:262:                time.sleep(120)
reana_job_controller/job_monitor.py:346:                time.sleep(120)
```